modules/apps/headless/headless-builder/headless-builder-web/src/main/resources/META-INF/resources/js/components/utils/urlUtil.ts

```diff
@@ -38,3 +38,15 @@ export function openEditURL({
 	newURLSearchParams.set(`_${portletId}_apiApplicationId`, id.toString());
 	window.location.search = newURLSearchParams.toString();
 }
+
+export function updateHistory({
+	navState,
+	portletId,
+}: {
+	navState: string;
+	portletId: string;
+}) {
+	const newURL = new URL(window.location.href);
+	newURL.searchParams.set(`_${portletId}_editAPIApplicationNav`, navState);
```

Comment:
@cgmonte Please, add a blank line between the definition of the const and its usage.